yaksh/urls.py

```diff
@@ -200,5 +200,9 @@
     url(r'^manage/preview_questionpaper/(?P<questionpaper_id>\d+)/$',
         views.preview_questionpaper, name="preview_questionpaper"),
     url(r'^manage/get_user_status/(?P<course_id>\d+)/(?P<student_id>\d+)/$',
-        views.get_user_data, name="get_user_data")
+        views.get_user_data, name="get_user_data"),
+    url(r'^manage/courses/download_course/(?P<course_id>\d+)/$',
+        views.download_course, name="download_course"),
+    url(r'^download_course/(?P<course_id>\d+)/$',
+        views.download_course, name="download_course"),
```

Comment:
Both urls are similar and they have same name!
Same name is fine if there is difference in the arguments.
Anyway here it wont harm as as nothing special is happening for the moderator.
And we try to use {% url %} tag or reserve match then in case of tie it will take the last url.
Its fine for now, in future we need to be careful.